<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fff;
color: #000;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_NavBar">
      <property name="styleSheet">
       <string notr="true">border-bottom: 2px solid #f5f5f5;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../assets/assets.qrc">:/icons/images/icons/logo.png</pixmap>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Roboto Medium</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}</string>
         </property>
         <property name="text">
          <string>EmoSense</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::PlainText</enum>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_Home">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_RealTimeEmotion">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Real-time Emotion Recognition</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_EmotionAnalytics">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Emotion Analytics</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Settings">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_AboutUs">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>About Us</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Login">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Logout">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Close">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #FFF;
    color: #000;
    border: none;
}

QLabel:hover {
    color: #011BA1;
}</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
